File: PRAK304-2310817210019-NaufalElyzar.py
n = int(input())

if 0 < n < 10:
    print("Satuan")
elif 10 <= n < 20:
    print("Belasan")
elif 20 <= n < 100:
    print("Puluhan")
elif n == 0:
    print("Nol")
elif n > 99:
    print("Anda Menginput Melebihi Limit Bilangan")

File: PRAK304-2310817210019-NaufalElyzar.c
    #include <stdio.h>

    int main() {
        int n;
        scanf("%d", &n);
        
        if (n > 0 && n < 10){
            printf("Satuan");
        }
        else if (n > 10 && n < 20){
            printf("Belasan");
        }
        else if (n > 20 && n < 100){
            printf("Puluhan");
        }
        else if (n == 0){
            printf("Nol");
        }
        else if (n > 99){
            printf("Anda Menginput Melebihi Limit Bilangan");
        }

        return 0;
    }
